src/ui/landing/components/landing.tsx

```diff
@@ -139,6 +139,10 @@ export class LandingComponent extends React.Component<Props, State> {
     })
   }
 
+  private handleButtonTap = () => {
```

Comment:
This is here so we can programatically trigger it from within tests, since simulating a click on a material ui button is not that consistent.
We are currently evaluating if this makes sense, because this is something that could be taken care of in integration tests.